### ISSUE TITLE
Do not log account passwords

### ISSUE THREAD

**Issue by pfoppe** [Esri/arcgis-powershell-dsc#170]
In reviewing results of an execution, the **ArcGISConfigure-<dt>-Verbose.txt** file has passwords logged to the file.  This is generally not a security best practice and logging passwords should be avoided.  

Here is a sample (redacted): 

> [SERVER]:                            [[ArcGIS_Service_Account]ArcGIS_DataStore_RunAs_Account] RunAsAccount Username:- DOMAIN\USERNAME
> [SERVER]:                            [[ArcGIS_Service_Account]ArcGIS_DataStore_RunAs_Account] Install Dir for ArcGIS Data Store is C:\Program Files\ArcGIS\DataStore\
> [SERVER]:                            [[ArcGIS_Service_Account]ArcGIS_DataStore_RunAs_Account] Just Checking : - /username DOMAIN\USERNAME /password "**CLEAR_TEXT_PASSWORD**"

**Comment by shailesh91** [comment]
@pfoppe this is an aberration. we will fix this in the upcoming release.

**Comment by pfoppe** [comment]
Thanks!

**Comment by spitzerr** [comment]
@scma-esrich FYI

**Comment by scma-esrich** [comment]
> 
> 
> @scma-esrich FYI

Thanks, @spitzerr. I already noticed that.

**Comment by pfoppe** [comment]
Please re-open.  I found another one on the data store upgrade at 10.7.1: 
```
...
[<SERVERNAME>]:                            [[ArcGIS_DataStoreUpgrade]DataStoreConfigUpgrade] Retrieved server token successfully
[<SERVERNAME>]:                            [[ArcGIS_DataStoreUpgrade]DataStoreConfigUpgrade] Relational Replication Role - PRIMARY
[<SERVERNAME>]:                            [[ArcGIS_DataStoreUpgrade]DataStoreConfigUpgrade] relational
[<SERVERNAME>]:                            [[ArcGIS_DataStoreUpgrade]DataStoreConfigUpgrade] C:\Program Files\ArcGIS\DataStore\tools\configuredatastore.bat https://<SERVERNAME><DOMAIN>:6443/arcgis <PORTAL_ADMIN_ACCOUNT> <PORTAL_CLEAR_TEXT_PASSWORD> D:\ArcGISDataStore --stores relational
[<SERVERNAME>]:                            [[ArcGIS_DataStoreUpgrade]DataStoreConfigUpgrade] Output of execution:- 
...
```
Thanks!

**Comment by pfoppe** [comment]
And Another...
```[<SERVER>]:                            [[ArcGIS_Install]DataStoreUpgrade] Installing Software using installer at \\server\share\InstallMedia\Esri\10.7.1\ArcGISDataStore\setup.exe and arguments /qb USER_NAME=<USERNAME> PASSWORD=<PASSWORD>```

**Comment by pfoppe** [comment]
And Another...
`[<SERVER>]:                            [[ArcGIS_Install]ServerUpgrade] Installing Software using installer at \\server\share\InstallMedia\Esri\10.7.1\ArcGISServer\setup.exe and arguments /qb USER_NAME=<USERNAME> PASSWORD=<PASSWORD>`

**Comment by shailesh91** [comment]
Fixed with v2.1.1!

**Comment by pfoppe** [comment]
I searched through the logs from a new v10.7.1 install with arcgis-powershell-dsc v2.1.1 and was unable to find any passwords being logged.  Thanks!
